### PR TITLE
Support option to not map events into an object

### LINKED ISSUE #21: Cleanup
Do this - https://github.com/hapijs/contrib/issues/31

### PR DESCRIPTION
**Pull request by geek**
Closes #21

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,10 +3,11 @@
   "version": "5.0.0",
   "repository": "git://github.com/hapijs/good-http",
   "description": "Http(s) broadcasting for Good process monitor",
-  "main": "index.js",
+  "main": "lib/index.js",
   "scripts": {
-    "test": "lab -m 5000 -t 100 -v -La code",
```
Comment (arb) [inline package.json]: Why the changes to the test scripts, they look fine to me.

Comment (geek) [inline package.json]: No need for the timeout from what I see

Comment (arb) [inline package.json]: You added tap that wasn't there before. Also, that timeout might not be needed locally, but it was needed for travis sometimes.
